patches/buildroot/0001-disable-shared-for-host-builds-of-gmp-isl-mpc-mpfr.patch

```diff
@@ -26,7 +26,7 @@ index 51f5cbce4f..01677cd82b 100644
  HOST_CXXFLAGS += $(HOST_CFLAGS)
 -HOST_LDFLAGS  += -L$(HOST_DIR)/lib -Wl,-rpath,$(HOST_DIR)/lib
 +HOST_LDFLAGS  += -L$(HOST_DIR)/lib
- 
+
```

Comment:
There was a lot of extraneous whitespace that my editor deleted. Is that a problem?